malaria-ios/User.swift

```diff
@@ -3,25 +3,18 @@ import CoreData
 
 class User: NSManagedObject {
```

Comment:
You have replicated User.swift files. Recheck the `git status`command for the list of files that were potentially added/modified. Using `git status` would find the extra `User.swift` file